composer.json

```diff
@@ -16,7 +16,7 @@
         "php": ">=7.1.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^7.5",
+        "phpunit/phpunit": ">=7.5",
```

Comment:
This should stay as the semantic compatibility tag. It's likely time to bump the phpunit dep up, but that's unrelated to this PR.